docs/workshop.md

```diff
@@ -0,0 +1,702 @@
+
+```json
+ {
+    "type": "WMS",
+    "name": "TopPlusOpen",
+    "url": "https://sgx.geodatenzentrum.de/wms_topplus_open?",
+    "layers": "web",
+    "lid": "topplusopen",
+    "attributions": "© Bundesamt für Kartographie und Geodäsie (2022)"
+  }
+```
+
+Note the new property `layers` which is specific for a `WMS` layer. Add this layer between the "airports" layer and the "positron" layer. Now our background shows more detail, but we cannot see our previous basemap anymore.
+
+For this problem Wegue has a inbuilt solution called "basemap switcher". Add the properties `"isBaseLayer": true` to both basemap layer. Wegue will then display a button on the top left side on the map where you can chose which baselayer should be shown.
+
+![basemap switcher](_media/workshop/basemap-switcher.png)
+
+This is how the `mapLayers` property should look like:
+
+```json
+ "mapLayers": [
+      {
+        "type": "VECTOR",
+        "format": "GeoJSON",
+        "lid": "airports",
+        "name": "Airports",
+        "style": {
+          "textIcon": "local_airport",
+          "font": "normal 35px Material Icons",
+          "fillColor": "black"
+        },
+        "url": "https://raw.githubusercontent.com/martynafford/natural-earth-geojson/90266457d82d3717e47313e72a921e31a087d2c4/50m/cultural/ne_50m_airports.json"
+      },
+      {
+        "type": "WMS",
+        "name": "TopPlusOpen",
+        "url": "https://sgx.geodatenzentrum.de/wms_topplus_open?",
+        "layers": "web",
+        "lid": "topplusopen",
+        "isBaseLayer": true,
+        "attributions": "© Bundesamt für Kartographie und Geodäsie (2022)"
+      },
+      {
+        "type": "XYZ",
+        "name": "Carto Positron",
+        "url": "https://basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png",
+        "lid": "positron",
+        "isBaseLayer": true,
+        "attributions": "Maps: © <a href='https://carto.com/attribution/' target='_blank'>CARTO</a> | <a href='https://openstreetmap.org/copyright' target='_blank'>© OpenStreetMap</a>-contributors | "
+      }
+    ]
+```
+
+## Use a module
+
+Wegue contains a lot of different modules that can be used. A frequently needed one is a layer switcher. This enables the user to switch layers on and off. To activate it, add this snippet below to your config file. Make sure it is located on the top level and also separated by an comma from other properties:
+
+```json
+  "modules": {
+      "wgu-layerlist": {
+        "target": "toolbar",
+        "win": "floating",
+        "icon": "layers"
+      }
+    }
+```
+Let's inspect what is happening here. The property `modules` is a JSON object with currently one object. However, there can be many more. Our module is called `wgu-layerlist` and has some properties by itself as well:
+
+- `"target": "toolbar"` adds the button for the layertree to the toolbar, alternatively it can be added to a menu using `"target": "menu"`
```

Comment:
```suggestion
- `"target": "toolbar"` adds the button for the layer list to the toolbar, alternatively it can be added to a menu using `"target": "menu"`
```